agent/bench-scripts/pbench-fio

```diff
@@ -603,28 +630,28 @@ function fio_run_job() {
 
 	if [ ! -z "$clients" ]; then
 		debug_log "creating directories on the clients"
-		for client in `echo $clients | sed -e s/,/" "/g`; do
+		for client in "${client_names[@]}"; do
 			ssh $ssh_opts $client mkdir -p $benchmark_results_dir &
 		done
 		wait
 		debug_log "opening port 8765 on firewall on the clients"
```

Comment:
The debug message needs to be fixed.

Reply:
Yes; the ports can be different, though, so the question is what the message should be.  One message, of the form "opening fio server port on the clients" or a separate message for each client?

Reply:
Perhaps something like:

```
if (( unique_ports )); then
    uniq_ports_text=" (unique ports per client)"
else
    uniq_ports_text=""
fi
debug_log "opening ports in firewalls of the clients${uniq_ports_text}"
```